README.md

```diff
@@ -1,21 +1,66 @@
+
+---
+## Configuration
+### **Driver Config**
+
+| Option | Type | Required | Default | Description |
+| :---: | :---: | :---: | :---: | :--- |
+| **enabled** | bool | no | true | Enable/Disable task driver. |
+| **stats_interval** | string | no | 1s | Interval for collecting `TaskStats` |
+
+### **Task Config**
+| Option | Type | Required | Default | Min. IIS Version | Description |
+| :---: | :---: | :---: | :---: | :---: | :--- |
+| **path** | string | yes | nil | 6.0 | Path to IIS Compatible website directory. |
+| **apppool_config_path** | string | no | nil | 6.0 | Path to App Pool XML Configuration File. |
+| **site_config_path** | string | no | nil | 6.0 | Path to Site XML Configuration File. |
+| **apppool_identity** | string | no | ApplicationPoolIdentity | 6.0 | Application Pool Identity e.g. ('SpecificUser', 'ApplicationPoolIdentity', etc..) |
+| **bindings** | block list | no | nil | 7.0 | This is needed to tie IIS Bindings to Nomad's `resources`->`network` ports to IIS as well as specify IIS Binding specific settings |
+### **Bindings Block Config**
+A `resource_port` OR a `port` must be provided. Due to a current limitation, we can not force at least 1 required option between multiple options. There are plans to revisit combining port options to improve UX.
+
+| Option | Type | Required | Default | Min. IIS Version | Description |
+| :---: | :---: | :---: | :---: | :---: | :--- |
+| **hostname** | string | no | nil | 7.0 | HostName attribute for a given binding. |
+| **ipaddress** | string | no | * | 7.0 | IPAddress attribute for a given binding. |
+| **resource_port** | string | no | nil | 7.0 | Tie a `resources`->`network` port label to the binding. This allows us to use a dynamic port given by Nomad. |
+| **port** | number | no | 0 | 7.0 | Port attribute for a given binding. This will overwrite `resource_port` settings. |
+| **type** | string | yes | nil | 7.0 | Type is the binding's protocol e.g. ('http', 'https', etc..) |
+| **cert_hash** | string | no | nil | 7.0 | Hash of a cert that exists prior to nomad allocating an IIS website. This **must** be set for SSL bindings. |
+For more info on IIS Bindings, you can go [here](https://docs.microsoft.com/en-us/iis/configuration/system.applicationhost/sites/site/bindings/binding)
+
+### **Environment Variables**
+Environment variables can be set like any other Nomad task via `env` or `template` stanzas. Environment variables are only supported for IIS 10.0+.
+
+### Meta Environment Variables
+These meta env vars do not persist to the process/task. They are pulled from the env var list that is passed to the IIS application pool. These do not require a minimal IIS version as they are not used as env vars by IIS.
+
+- `NOMAD_APPPOOL_USERNAME`
+  - Sets the UserName of an application pool and will override the Application Pool Identity to `SpecificUser`
```

Comment:
nitpick: `Username`

Reply:
Windows specifically uses this casing, so I kept it. I am a fan of Username as well.